modules/rosa-sts-setting-up-environment.adoc

```diff
@@ -73,30 +77,40 @@ $ rosa
 .Example output
 [source,terminal]
 ----
-Command line tool for ROSA.
+Command line tool for Red Hat OpenShift Service on AWS.
+For further documentation visit https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws
```

Comment:
Same as before, this is unnecessary.

Reply:
As above, this is just example output not a link